core/deployment/src/main/java/io/quarkus/deployment/dev/DevModeMain.java

```diff
@@ -176,6 +192,20 @@ private void silentDeleteFile(Path path) {
         }
     }
 
+    private void makeHardLinkWindowsFallback(Path link, Path dotEnvPath) throws IOException, InterruptedException {
+        Process process = new ProcessBuilder("cmd.exe", "/C", "mklink", "/H", link.toString(), dotEnvPath.toString())
+                .redirectOutput(new File("NUL"))
+                .redirectError(ProcessBuilder.Redirect.PIPE)
+                .start();
+        ByteArrayOutputStream errStream = new ByteArrayOutputStream();
+        ProcessUtil.streamErrorTo(new PrintStream(errStream), process);
+        int exitValue = process.waitFor();
```

Comment:
Put inside a try-finally block and call `process.destroy()` just in case

Reply:
Done. I wrapped everything after `start()`, just in case.